dll/win32/shell32/shellmenu/CMenuToolbars.h

```diff
@@ -152,16 +152,16 @@ class CMenuStaticToolbar :
     HRESULT SetMenu(HMENU hmenu, HWND hwnd, DWORD dwFlags);
     HRESULT GetMenu(HMENU *phmenu, HWND *phwnd, DWORD *pdwFlags);
 
-    virtual HRESULT FillToolbar(BOOL clearFirst=FALSE);
+    virtual HRESULT FillToolbar(BOOL clearFirst=FALSE) override;
 
 protected:
-    virtual HRESULT OnDeletingButton(const NMTOOLBAR * tb);
+    STDMETHOD(OnDeletingButton)(const NMTOOLBAR * tb) override;
 
-    virtual HRESULT InternalGetTooltip(INT iItem, INT index, DWORD_PTR dwData, LPWSTR pszText, INT cchTextMax);
-    virtual HRESULT InternalExecuteItem(INT iItem, INT index, DWORD_PTR dwData);
-    virtual HRESULT InternalPopupItem(INT iItem, INT index, DWORD_PTR dwData, BOOL keyInitiated);
-    virtual HRESULT InternalHasSubMenu(INT iItem, INT index, DWORD_PTR dwData);
-    virtual HRESULT InternalContextMenu(INT iItem, INT index, DWORD_PTR dwData, POINT pt);
+    STDMETHOD(InternalGetTooltip)(INT iItem, INT index, DWORD_PTR dwData, LPWSTR pszText, INT cchTextMax) override;
+    STDMETHOD(InternalExecuteItem)(INT iItem, INT index, DWORD_PTR dwData) override;
+    STDMETHOD(InternalPopupItem)(INT iItem, INT index, DWORD_PTR dwData, BOOL keyInitiated) override;
+    STDMETHOD(InternalHasSubMenu)(INT iItem, INT index, DWORD_PTR dwData) override;
+    STDMETHOD(InternalContextMenu)(INT iItem, INT index, DWORD_PTR dwData, POINT pt) override;
```

Comment:
Not sure whether it would be better to reserve the "STDMETHOD" thing _**just**_ for the overriden interface functions.
Here these are private/protected helpers of the class, so I'm not fully convinced of the change.

Reply:
Fixed.